packages/components/src/components/hds/card/container.ts

```diff
@@ -5,31 +5,45 @@
 
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
+import { BackgroundValues, LevelValues, OverflowValues } from './types.ts'
+import type { Background, Level, Overflow } from './types.ts';
+
+export const DEFAULT_LEVEL = LevelValues.Base;
+export const DEFAULT_BACKGROUND = BackgroundValues.NeutralPrimary;
+export const DEFAULT_OVERFLOW = OverflowValues.Visible;
+export const AVAILABLE_LEVELS: string[] = Object.values(LevelValues);
+export const AVAILABLE_BACKGROUNDS: string[] = Object.values(BackgroundValues);
+export const AVAILABLE_OVERFLOWS: string[] = Object.values(OverflowValues);
+
+export interface HdsCardContainerSignature {
+  Args: {
```

Comment:
we need to mark optional args, otherwise consumers will get type error that all args are required